docs/source/developers/release.rst

```diff
@@ -36,8 +39,8 @@ Preparing for the release
 =========================
 
 Before creating a source release, the Release Manager must ensure that any
-resolved JIRAs have the appropriate Fix Version set so that the changelog is
-generated properly.
+resolved GitHub issues and any MINOR pull requests have the appropriate
```

Comment:
MINOR pull requests don't require the milestone for the changelog to be generated correctly. The changelog goes over the issues on GH to validate no issues are tagged but missing from commits but for minor only the commits on the branch are required.